proms-resources/backend/src/main/java/io/uhndata/cards/proms/api/PatientAuthConfigUtils.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.uhndata.cards.proms.api;
+
+/**
+ * Basic utilities for grabbing configuration details from the patient identification node.
```

Comment:
How the configuration is stored is an implementation detail. We may switch this to OSGI config in the future, or there may be multiple configurations for different endpoints. The important thing is that this is the patient authentication configuration.

```suggestion
 * Configuration for how the patient can access the patient-facing UI.
```